sabnzbd/config.py

```diff
@@ -93,6 +93,7 @@ def get_dict(self, safe=False):
 
     def set_dict(self, dict):
         """ Set value based on dictionary """
+        # TODO: Review if `dict` is actually correct or typo
```

Comment:
dict is a reserved word, and were passing an object to overwrite another object. so it just screams at me in my ide about it.. why i put a note to review :)